src/components/QRcode.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+<div>
+  <canvas id="canvas"></canvas>
+</div>
+</template>
+<script>
+import QRCode from 'qrcode'
+export default {
+  props: {
+    qrcode: [String, Object],
+  },
+  mounted() {
+    var canvas = this.$el.querySelector('#canvas')
+    QRCode.toCanvas(canvas, this.qrcode, function(error) {
+      if (error) console.error(error)
```

Comment:
use proper warning/error alert, dont use console.log
console.log is just for testing when in development